config/ruby-1.8.yml

```diff
@@ -1,8 +0,0 @@
-inherit_from: ./ruby-1.9.yml
```

Comment:
Apologies for being dense—what necessitated the pre-2.2 configs being deleted? My understanding was rubocop had to be _run_ under 2.2 or later, but the way standard works is that while it must _execute_ 2.2, it can detect and lint 1.8 _code listings_

Reply:
I might be missing something crucial here. Are you saying that this code needs to be able to lint code outside of its own runtime? i.e. I run `rubocop path/to/some/dir` and inside `path/to/some/dir` there is ruby 1.8 code but from my working directory where I ran rubocop, it uses ruby >= 2.3? 

We still need whatever is running this `standard` to be >= Ruby 2.3 due to the commit I linked in this PR description.

Reply:
Yes, exactly. Running rubocop under MRI 2.2, standard can lint codebases that support 1.8, that's what all these special one-off configs are here for.

Reply:
Ah, that makes a lot more sense. I didn't think of that use case, thanks!